exercises/sieve/test/test_sieve.c

```diff
@@ -0,0 +1,114 @@
+#include <string.h>
+#include "vendor/unity.h"
+#include "../src/sieve.h"
+
+primesArray_t result_array;
+
+// Individual Tests
+void test_no_primes_under_two(void)
+{
+   const unsigned int limit = 1;
+   const primesArray_t expected_prime_array = { 0 };
```

Comment:
you can't do that...  it has to have an entry or no initializer...   since I want to init the expected values and leave the rest as 0 to support memcmp  -- this was the easiest way as it will make everything 0